packages/auth/package.json

```diff
@@ -29,6 +29,7 @@
     "node-mocks-http": "^1.11.0",
     "objection": "^3.0.1",
     "objection-db-errors": "^1.1.2",
+    "open-payments": "workspace:*",
```

Comment:
```
pnpm --filter auth add -P open-payments --workspace
```
gave me this instead of:
https://github.com/interledger/rafiki/blob/e796dedeee7c3aedd43c5c977ce83f420ea8667c/packages/backend/package.json#L84